test/unit-tests/libraries/Message.t.sol

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.9;
+
+/// library imports
+import {Test, Vm} from "forge-std/Test.sol";
+
+/// local imports
+import "src/libraries/Message.sol";
+
+/// @dev is a helper function to test library contracts
+contract MessageHelper {
+    using MessageLibrary for MessageLibrary.Message;
+
+    function computeMsgId(MessageLibrary.Message memory _message) external pure returns (bytes32) {
+        return _message.computeMsgId();
+    }
+
+    function extractExecutionParams(MessageLibrary.Message memory _message)
+        external
+        pure
+        returns (MessageLibrary.MessageExecutionParams memory)
+    {
+        return _message.extractExecutionParams();
+    }
+
+    function computeExecutionParamsHash(MessageLibrary.MessageExecutionParams memory _params)
+        external
+        pure
+        returns (bytes32)
+    {
+        return MessageLibrary.computeExecutionParamsHash(_params);
+    }
+
+    function computeExecutionParamsHashFromMessage(MessageLibrary.Message memory _message)
+        external
+        pure
+        returns (bytes32)
+    {
+        return _message.computeExecutionParamsHash();
+    }
+}
+
+contract MessageLibraryTest is Test {
+    MessageHelper messageHelper;
+
+    function setUp() public {
+        messageHelper = new MessageHelper();
+    }
+
+    /// @dev tests computation of message id
+    function testComputeMsgId() public {
```

Comment:
can we change the testing naming style to be consistent with other unit tests (i.e. underscore separated) instead of the mixed camel case used here?  The implicit convention we've adopted thus far is the former for unit tests and the latter for integration tests.